source/common/http/http1/codec_impl.cc

```diff
@@ -861,6 +864,58 @@ void ConnectionImpl::onResetStreamBase(StreamResetReason reason) {
   onResetStream(reason);
 }
 
+void ConnectionImpl::dumpState(std::ostream& os, int indent_level) const {
+  const char* spaces = spacesForLevel(indent_level);
+  // Dump all bool to provide context
+  os << spaces << "Http1::ConnectionImpl " << this << DUMP_MEMBER(dispatching_)
+     << DUMP_MEMBER(dispatching_slice_already_drained_) << DUMP_MEMBER(reset_stream_called_)
+     << DUMP_MEMBER(handling_upgrade_) << DUMP_MEMBER(deferred_end_stream_headers_)
+     << DUMP_MEMBER(strict_1xx_and_204_headers_) << DUMP_MEMBER(processing_trailers_);
+
+  // Dump the first slice of the dispatching buffer and buffered_body if
+  // applicable.
+  auto dumpBuffer = [](auto* instance) {
+    auto slice = instance->frontSlice();
+    return absl::string_view(static_cast<const char*>(slice.mem_), slice.len_);
+  };
+
+  os << DUMP_NULLABLE_MEMBER(current_dispatching_buffer_, dumpBuffer(current_dispatching_buffer_));
+  os << DUMP_MEMBER(buffered_body_, dumpBuffer(&buffered_body_));
```

Comment:
I think we shoud consider dumping just buffered_body_.length() or omit this field, the body contents is not that relevant. The current_dispatching_buffer_ seems more relevant.